### PR TITLE
Adds descriptions to all cargo orders!

### PR DESCRIPTION
**Pull request by MrDoomBringer**
i sacrifice sweet sweet gbp to bring you, the players, nice fluffy text

:cl: MrDoomBringer
add: Orderable supplies in cargo now all have descriptions! The station's overall FLAVORFUL_TEXT stat has gone up by nearly 2% as a result.
/:cl:

[why]: so people know what they waste their credits on.
also deletes some duplicate orders from #35970

### CODE REVIEW

**Comment by MrDoomBringer** [comment]
fuck im retarded gimme a bit to rebuild tgui and erase my mistakes

**Comment by 81Denton** [comment]
How many GBP do I have to pay you to do the same for the mining vendor?

**Comment by MrDoomBringer** [comment]
@81Denton 
image

**Comment by 81Denton** [comment]
One MILLION GBP?

**Comment by MrDoomBringer** [comment]
yes

File: code/modules/cargo/packs.dm [hunk 1]
```diff
@@ -1695,6 +1744,7 @@
 
 /datum/supply_pack/misc/foamforce/bonus
 	name = "Foam Force Pistols Crate"
+	desc = "psst..hey bud... remember those old foam force pistols that got discontinued for being too cool? well i got two of those right here with your name on em. I'll even throw in a spare mag for each, waddya say?"
```
Comment (F-OS) [inline code/modules/cargo/packs.dm]: Capitalize well, anything after a !, ?, or . Is a new sentence, so capitalize that shit.

**Comment by 81Denton** [comment]
Holy shit you actually wrote individual descriptions for every single pack

Someone give this man a medal!

File: code/modules/cargo/packs.dm [hunk 2]
```diff
@@ -609,6 +663,7 @@
 	
 /datum/supply_pack/engineering/engine
 	name = "Emitter Crate"
+	desc = "Useful for powering forcefield generators while destroying locked crates and intruders alike. Contains two high-powered energy emitters."
```
Comment (81Denton) [inline code/modules/cargo/packs.dm]: This one's missing the "requires CE access" line.
Also holy shit, props for writing all these descriptions.

**Comment by AnturK** [comment]
Diff looks broken.

**Comment by MrDoomBringer** [comment]
ahhhhahhahahah fuck

File: code/modules/cargo/console.dm [hunk 3]
```diff
@@ -77,7 +77,8 @@
 		data["supplies"][P.group]["packs"] += list(list(
 			"name" = P.name,
 			"cost" = P.cost,
-			"id" = pack
+			"id" = pack,
+			"desc" = P.desc ? P.desc : P.name // If there is a description, use it. Otherwise use the pack's name.
```
Comment (duncathan) [inline code/modules/cargo/console.dm]: replace this with `"desc" = P.desc || P.name` for a more concise version of the same behavior

File: code/modules/cargo/expressconsole.dm [hunk 4]
```diff
@@ -58,7 +58,8 @@
 		meme_pack_data[P.group]["packs"] += list(list(
 			"name" = P.name,
 			"cost" = P.cost,
-			"id" = pack
+			"id" = pack,
+			"desc" = P.desc ? P.desc : P.name // If there is a description, use it. Otherwise use the pack's name.
```
Comment (duncathan) [inline code/modules/cargo/expressconsole.dm]: same here

File: code/modules/cargo/packs.dm [hunk 5]
```diff
@@ -67,6 +70,7 @@
 
 /datum/supply_pack/emergency/equipment
 	name = "Emergency Bot/Internals Crate"
+	desc = "Explosions got you down? These supplies are guarenteed to patch up holes, in stations and people alike! Comes with two floorbots, two medbots, five oxygen masks and five small oxygen tanks."
```
Comment (duncathan) [inline code/modules/cargo/packs.dm]: guaranteed*

File: code/modules/cargo/packs.dm [hunk 6]
```diff
@@ -212,6 +225,7 @@
 
 /datum/supply_pack/emergency/weedcontrol
 	name = "Weed Control Crate"
+	desc = "Keep those invasive species OUT. Contains a scythe, gasmask, and two anti-weed chemical grenades. Warrenty void if used on cannabis. Requires Hydroponics access to open."
```
Comment (duncathan) [inline code/modules/cargo/packs.dm]: replace cannabis with ambrosia imo

File: code/modules/cargo/packs.dm [hunk 7]
```diff
@@ -359,6 +386,7 @@
 
 /datum/supply_pack/security/armory/bulletarmor
 	name = "Bulletproof Armor Crate"
+	desc = "Contains three sets of bulletproof armor. Guarenteed to reduce a bullet's stoping power by over half. Requires Armory access to open."
```
Comment (duncathan) [inline code/modules/cargo/packs.dm]: stopping*

File: code/modules/cargo/packs.dm [hunk 8]
```diff
@@ -367,6 +395,7 @@
 
 /datum/supply_pack/security/armory/combatknives
 	name = "Combat Knives Crate"
+	desc = "Contains three sharpened combat knives. Each knife guarenteed to fit snugly inside any Nanotrasen-standard boot. Requires Armory access to open."
```
Comment (duncathan) [inline code/modules/cargo/packs.dm]: guaranteed*

File: code/modules/cargo/packs.dm [hunk 9]
```diff
@@ -375,6 +404,7 @@
 
 /datum/supply_pack/security/armory/ballistic
 	name = "Combat Shotguns Crate"
+	desc = "For when the enemy absolutely needs to be replaced with lead. Contains three Aussec-designed Combat Shotguns, and three Shotgun Bandoleirs. Requires Armory access to open."
```
Comment (duncathan) [inline code/modules/cargo/packs.dm]: Bandoliers* or Bandoleers*

File: code/modules/cargo/packs.dm [hunk 10]
```diff
@@ -441,6 +478,7 @@
 
 /datum/supply_pack/security/armory/riotarmor
 	name = "Riot Armor Crate"
+	desc = "Contains three sets of heavy body armor. Advanced padding protects against close-ranged weaponry, making meele attacks feel only half as potent to the user. Requires Armory access to open."
```
Comment (duncathan) [inline code/modules/cargo/packs.dm]: melee*

File: code/modules/cargo/packs.dm [hunk 11]
```diff
@@ -481,13 +522,15 @@
...
 	name = "WT-550 Auto Rifle Crate"
+	desc = "Contains two high-powered, semiautomatic rifles chambered in 4.6x30mm. Requires Armory access to open."
 	cost = 3500
 	contains = list(/obj/item/gun/ballistic/automatic/wt550,
 					/obj/item/gun/ballistic/automatic/wt550)
 	crate_name = "auto rifle crate"
 
 /datum/supply_pack/security/armory/wt550ammo
 	name = "WT-550 Auto Rifle Ammo Crate"
+	desc = "Contains four 20-round magazines for the WT-550 Auto Rifle. Each magazines is designed to facilitate rapid tactical reloads. Requires Armory access to open."
```
Comment (duncathan) [inline code/modules/cargo/packs.dm]: Each magazine*

File: code/modules/cargo/packs.dm [hunk 12]
```diff
@@ -506,13 +549,15 @@
...
 	name = "Anti-breach Shield Projector Crate"
+	desc = "Hull breaches again? Say no more with the Nanotrasen Anti-Breach Shield Projector! Uses forcefield technology to keep the air in, and the space out. Contains two shield projectors."
 	cost = 2500
 	contains = list(/obj/machinery/shieldgen,
 					/obj/machinery/shieldgen)
 	crate_name = "anti-breach shield projector crate"
 
 /datum/supply_pack/engineering/engiequipment
 	name = "Engineering Gear Crate"
+	desc = "Gear up with three toolbelts, high-visibility vests, welding helmets, hardhats, and two meson goggles!"
```
Comment (duncathan) [inline code/modules/cargo/packs.dm]: two pairs of meson goggles*

File: code/modules/cargo/packs.dm [hunk 13]
```diff
@@ -532,32 +577,36 @@
...
 	name = "NT-75 Electromagnetic Power Inducers Crate"
+	desc = "No rechargers? No problem, with the NT-75 EPI, you can recharge any standard cell-based equipment anytime, anywhere. Contains two Inducers."
 	cost = 2000
 	contains = list(/obj/item/inducer/sci {cell_type = /obj/item/stock_parts/cell/inducer_supply; opened = 0}, /obj/item/inducer/sci {cell_type = /obj/item/stock_parts/cell/inducer_supply; opened = 0}) //FALSE doesn't work in modified type paths apparently.
 	crate_name = "inducer crate"
 	crate_type = /obj/structure/closet/crate/engineering/electrical
 
 /datum/supply_pack/engineering/pacman
 	name = "P.A.C.M.A.N Generator Crate"
+	desc = "Engineers cant set up the engine? Not an issue for you, once you get your hands on this P.A.C.M.A.N. Generator! Takes in plasma and spits out sweet sweet energy."
```
Comment (duncathan) [inline code/modules/cargo/packs.dm]: can't*

File: code/modules/cargo/packs.dm [hunk 14]
```diff
@@ -576,30 +626,34 @@
 
 /datum/supply_pack/engineering/tools
 	name = "Toolbox Crate"
+	desc = "Any robust spaceman is never far from their trusty toolbox. Contains three electircal toolboxes and three mechanical toolboxes."
```
Comment (duncathan) [inline code/modules/cargo/packs.dm]: electrical*

File: code/modules/cargo/packs.dm [hunk 15]
```diff
@@ -625,13 +680,15 @@
...
 	name = "Field Generator Crate"
+	desc = "Typically the only thing standing between the station and a messy death. Powered by emitters. Contains two field generators."
 	cost = 1500
 	contains = list(/obj/machinery/field/generator,
 					/obj/machinery/field/generator)
 	crate_name = "field generator crate"
 
 /datum/supply_pack/engineering/grounding_rods
 	name = "Grounding Rod Crate"
+	desc = "Four grounding rods guarenteed to keep any uppity tesla's lightning under control."
```
Comment (duncathan) [inline code/modules/cargo/packs.dm]: guaranteed*

File: code/modules/cargo/packs.dm [hunk 16]
```diff
@@ -707,12 +769,14 @@
...
 /datum/supply_pack/engineering/engine/tesla_gen
 	name = "Tesla Generator Crate"
+	desc = "The key to unlocking the power of the Tesla energy ball. Particle Accelerator not included."
 	cost = 5000
 	contains = list(/obj/machinery/the_singularitygen/tesla)
 	crate_name = "tesla generator crate"
 
 /datum/supply_pack/engineering/bsa
 	name = "Bluespace Artillery Parts"
+	desc = "The pride of Nanotrasen Naval Command. The legendary Bluespace Artillery Cannon is a devastating feat of human engineering and testimate to wartime determination. Highly advanced research is required for proper construction. "
```
Comment (duncathan) [inline code/modules/cargo/packs.dm]: testament*

File: code/modules/cargo/packs.dm [hunk 17]
```diff
@@ -777,54 +845,63 @@
...
 /datum/supply_pack/materials/wood50
 	name = "50 Wood Planks"
+	desc = "Turn cargo's boring metal groundwork into beautiful panelled flooring and much more with fifty wooden planks!"
 	cost = 2000
 	contains = list(/obj/item/stack/sheet/mineral/wood/fifty)
 	crate_name = "wood planks crate"
 
 /datum/supply_pack/materials/bz
 	name = "BZ Canister Crate"
+	desc = "Contains a canister of Benzene. Requires Toxins access to open."
```
Comment (duncathan) [inline code/modules/cargo/packs.dm]: It's actually just called BZ everywhere else, do that here too

File: code/modules/cargo/packs.dm [hunk 18]
```diff
@@ -855,34 +935,39 @@
...
 	name = "Fuel Tank Crate"
+	desc = "Contains a welding fuel tank. Caution, highly flammable."
 	cost = 800
 	contains = list(/obj/structure/reagent_dispensers/fueltank)
 	crate_name = "fuel tank crate"
 	crate_type = /obj/structure/closet/crate/large
 
 /datum/supply_pack/materials/watertank
 	name = "Water Tank Crate"
+	desc = "Contains a tank of dihydrogen monoxide...sounds dangerous."
```
Comment (duncathan) [inline code/modules/cargo/packs.dm]: add a space after the ellipse

File: code/modules/cargo/packs.dm [hunk 19]
```diff
@@ -1042,6 +1139,7 @@
 
 /datum/supply_pack/science/robotics/mecha_ripley
 	name = "Circuit Crate (Ripley APLU)"
+	desc = "Rip apart rocks and aliens alike with the Ripley APLU. Contains the Main Ripley control board, as well as the Ripley Peripherals board. Requires Robotics access to open."
```
Comment (duncathan) [inline code/modules/cargo/packs.dm]: replace aliens with xenomorphs

File: code/modules/cargo/packs.dm [hunk 20]
```diff
@@ -1052,6 +1150,7 @@
 
 /datum/supply_pack/science/plasma
 	name = "Plasma Assembly Crate"
+	desc = "Everything you need to burn something to the ground, this contains three plasma assembly sets. Each set contains a plasma tank, igniter, proximity sensor, and timer! Warrenty void if exposed to high temperatures. Requires Toxins access to open."
```
Comment (duncathan) [inline code/modules/cargo/packs.dm]: Warranty*

File: code/modules/cargo/packs.dm [hunk 21]
```diff
@@ -1085,6 +1185,7 @@
 
 /datum/supply_pack/science/shieldwalls
 	name = "Shield Generator Crate"
+	desc = "These high powered Shield Wall Generators are guarenteed to keep any unwanted lifeforms on the outside, where they belong! Contains four shield wall generators. Requires Teleporter access to open."
```
Comment (duncathan) [inline code/modules/cargo/packs.dm]: guaranteed*

File: code/modules/cargo/packs.dm [hunk 22]
```diff
@@ -1122,21 +1225,9 @@
...
-					/obj/item/honey_frame,
-					/obj/item/queen_bee/bought,
-					/obj/item/clothing/head/beekeeper_head,
-					/obj/item/clothing/suit/beekeeper_suit,
-					/obj/item/melee/flyswatter)
-	crate_name = "beekeeping starter crate"
-
 /datum/supply_pack/organic/hydroponics/beekeeping_suits
 	name = "Beekeeper Suit Crate"
+	desc = "Bee buisiness booming? Better be benevolent and boost botany by bestowing bi-Beekeeper-suits! Contains two beekeeper suits and matching headwear."
```
Comment (duncathan) [inline code/modules/cargo/packs.dm]: business*

File: code/modules/cargo/packs.dm [hunk 23]
```diff
@@ -1162,141 +1254,33 @@
...
+					/obj/item/honey_frame,
+					/obj/item/queen_bee/bought,
+					/obj/item/clothing/head/beekeeper_head,
+					/obj/item/clothing/suit/beekeeper_suit,
+					/obj/item/melee/flyswatter)
+	crate_name = "beekeeping starter crate"
+
 /datum/supply_pack/organic/cream_piee
 	name = "High-yield Clown-grade Cream Pie Crate"
+	desc = "Designed by Aussec's Advanced Warfare Research Division, these high-yield, Clown-grade cream pies are powered by a synergy of performance and efficiency. Guarenteed to provide maximum results."
```
Comment (duncathan) [inline code/modules/cargo/packs.dm]: Guaranteed*

File: code/modules/cargo/packs.dm [hunk 24]
```diff
@@ -1162,141 +1254,33 @@
...
-	crate_name = "entomology samples crate"
-
-/datum/supply_pack/organic/critter/butterfly/generate()
-	. = ..()
-	for(var/i in 1 to 49)
-		new /mob/living/simple_animal/butterfly(.)
-
 /datum/supply_pack/organic/hydroponics
 	name = "Hydroponics Crate"
+	desc = "Supplies for growing a great garden! Contains two bottles of ammonia, two plant-b-gone spray bottles, a hatchet, cultivator, plant analyzer, as well as a pair of leather gloves and a botanist's apron."
```
Comment (duncathan) [inline code/modules/cargo/packs.dm]: Plant-B-Gone*

File: code/modules/cargo/packs.dm [hunk 25]
```diff
@@ -1375,6 +1335,7 @@
 
 /datum/supply_pack/organic/pizza
 	name = "Pizza Crate"
+	desc = "Best prices on this side of the galaxy. All deliveries are guarenteed to be 99% anomaly-free!"
```
Comment (duncathan) [inline code/modules/cargo/packs.dm]: guaranteed*

File: code/modules/cargo/packs.dm [hunk 26]
```diff
@@ -1408,6 +1369,7 @@
 
 /datum/supply_pack/organic/potted_plants
 	name = "Potted Plants Crate"
+	desc = "Spruce up the station with these lovley plants! Contains a random assortment of five potted plants from Nanotrasen's potted plant research division. Warrenty void if thrown."
```
Comment (duncathan) [inline code/modules/cargo/packs.dm]: lovely*

Comment (duncathan) [reply code/modules/cargo/packs.dm]: Warranty*

File: code/modules/cargo/packs.dm [hunk 27]
```diff
@@ -1457,6 +1421,7 @@
 
 /datum/supply_pack/organic/critter/butterfly
 	name = "Butterflies Crate"
+	desc = "Not a very dangerous insect, but they do give off a better image than say, flies or cockroaches."//is that a motherfucking worm reference
```
Comment (duncathan) [inline code/modules/cargo/packs.dm]: ... than, say, flies ...

File: code/modules/cargo/packs.dm [hunk 28]
```diff
@@ -1484,12 +1450,27 @@
 
 /datum/supply_pack/organic/critter/chick
 	name = "Chicken Crate"
+	desc = "The chiken goes bwaak!"
```
Comment (duncathan) [inline code/modules/cargo/packs.dm]: ok come on man how'd you fuck this one up

File: code/modules/cargo/packs.dm [hunk 29]
```diff
@@ -1484,12 +1450,27 @@
...
 	cost = 2000
 	contains = list( /mob/living/simple_animal/chick)
 	crate_name = "chicken crate"
 
+/datum/supply_pack/organic/critter/crab
+	name = "Crab Rocket"
+	desc = "CRAAAAAAB ROCKET. CRAB ROCKET. CRAB ROCKET.  CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB ROCKET. CRAFT. ROCKET. BUY. CRAFT ROCKET. CRAB ROOOCKET. CRAB ROOOOCKET. CRAB CRAB CRAB CRAB CRAB CRAB CRAB CRAB ROOOOOOOOOOOOOOOOOOOOOOCK EEEEEEEEEEEEEEEEEEEEEEEEE EEEETTTTTTTTTTTTAAAAAAAAA AAAHHHHHHHHHHHHH. CRAB ROCKET. CRAAAB ROCKEEEEEEEEEGGGGHHHHTT CRAB CRAB CRAABROCKET CRAB ROCKEEEET."//fun fact: i actually spent like 10 minutes and transcribed the entire video.
```
Comment (duncathan) [inline code/modules/cargo/packs.dm]: there's an extra space after the third period

File: code/modules/cargo/packs.dm [hunk 30]
```diff
@@ -1504,40 +1485,46 @@
...
 	name = "Fox Crate"
+	desc = "The fox goes...? Comes with a collar!"//what does the fox say
 	cost = 5000
 	contains = list(/mob/living/simple_animal/pet/fox,
 					/obj/item/clothing/neck/petcollar)
 	crate_name = "fox crate"
 
 /datum/supply_pack/organic/critter/goat
 	name = "Goat Crate"
+	desc = "The goat goes baa! Warrenty void if used as a replacement for Pete."
```
Comment (duncathan) [inline code/modules/cargo/packs.dm]: Warranty*

File: code/modules/cargo/packs.dm [hunk 31]
```diff
@@ -1504,40 +1485,46 @@
...
 	cost = 2500
 	contains = list(/mob/living/simple_animal/hostile/retaliate/goat)
 	crate_name = "goat crate"
 
 /datum/supply_pack/organic/critter/pug
 	name = "Pug Crate"
+	desc = "Like a normal dog, but...squished. Comes with a nice collar!"
```
Comment (duncathan) [inline code/modules/cargo/packs.dm]: but... squished.

File: code/modules/cargo/packs.dm [hunk 32]
```diff
@@ -1604,13 +1596,15 @@
 
 /datum/supply_pack/misc/bicycle
 	name = "Bicycle"
+	desc = "Nanotrasen reminds all employees to never toy with powers outside their control"
```
Comment (duncathan) [inline code/modules/cargo/packs.dm]: add a period

File: code/modules/cargo/packs.dm [hunk 33]
```diff
@@ -1627,6 +1671,7 @@
 
 /datum/supply_pack/misc/paper
 	name = "Bureaucracy Crate"
+	desc = "High stacks of papers on your desk Are a big problem - make it Pea-sized with these bureacratic supplies! Contains six pens, some camera film, hand labeler supplies, a paper bin, three folders, two clipboards and two stamps."//that was too forced
```
[the remark this answers is not in the document]
Comment (duncathan) [reply code/modules/cargo/packs.dm]: Are Pea

File: code/modules/cargo/packs.dm [hunk 34]
```diff
@@ -1658,12 +1654,14 @@
...
 /datum/supply_pack/misc/fountainpens
 	name = "Calligraphy Crate"
+	desc = "Sign death warrents in style with these seven executive fountain pens."
 	cost = 700
 	contains = list(/obj/item/storage/box/fountainpens)
 	crate_type = /obj/structure/closet/crate/wooden
 
 /datum/supply_pack/misc/randomised/contraband
 	name = "Contraband Crate"
+	desc = "psst.. bud... want some contraband? I can get you a poster, some nice cigs, bling, even some ambrosia deus...you know, the good stuff. Just keep it away from the cops, kay?"
```
Comment (duncathan) [inline code/modules/cargo/packs.dm]: Psst...*

File: code/modules/cargo/packs.dm [hunk 35]
```diff
@@ -1702,6 +1702,7 @@
 
 /datum/supply_pack/misc/foamforce/bonus
 	name = "Foam Force Pistols Crate"
+	desc = "psst..hey bud... remember those old foam force pistols that got discontinued for being too cool? Well I got two of those right here with your name on em. I'll even throw in a spare mag for each, waddya say?"
```
Comment (duncathan) [inline code/modules/cargo/packs.dm]: Psst... *

File: code/modules/cargo/packs.dm [hunk 36]
```diff
@@ -1783,20 +1790,23 @@
...
 	name = "Laser Tag Firing Pins Crate"
+	desc = "Three laser tag firing pins used in laser-tag units to ensure users are wearing their vests."
 	cost = 3000
 	contraband = TRUE
 	contains = list(/obj/item/storage/box/lasertagpins)
 	crate_name = "laser tag crate"
 
 /datum/supply_pack/misc/mule
 	name = "MULEbot Crate"
+	desc = "Pink-haired Quartermaster not doing her job? Replace them with this tireless worker, today!"
```
Comment (duncathan) [inline code/modules/cargo/packs.dm]: Replace her*

File: code/modules/cargo/packs.dm [hunk 37]
```diff
@@ -1810,6 +1820,7 @@
 
 /datum/supply_pack/misc/janitor/lightbulbs
 	name = "Replacement Lights"
+	desc = "May the light of Aether shine upon this station! Or at least, the light of forty two light tubes and twenty one light bulbs."
```
Comment (duncathan) [inline code/modules/cargo/packs.dm]: is this a metroid prime 2 reference if so i love you

File: code/modules/cargo/packs.dm [hunk 38]
```diff
@@ -1870,6 +1885,7 @@
 
 /datum/supply_pack/misc/costume_original
 	name = "Original Costume Crate"
+	desc = "Reenact shakespearean plays with this assortment of outfits. Contains eight different costumes!"
```
Comment (duncathan) [inline code/modules/cargo/packs.dm]: Shakespearean*

File: code/modules/cargo/packs.dm [hunk 39]
```diff
@@ -1920,6 +1937,7 @@
 
 /datum/supply_pack/misc/wizard
 	name = "Wizard Costume Crate"
+	desc = "Pretend to join the Wizard Federation with this full wizard outfit! Nanotrasen would like to remind it's employees that actually joining the Wizard Federation is subject to termination of job and life."
```
Comment (duncathan) [inline code/modules/cargo/packs.dm]: its*

File: code/modules/cargo/packs.dm [hunk 40]
```diff
@@ -1962,6 +1981,7 @@
 
 /datum/supply_pack/misc/randomised/toys
 	name = "Toy Crate"
+	desc = "Who cares about pride and accomplishment? Skip the gaming and get straight to the sweet rewards with this product! Contains five random toys. Warrenty void if used to prank research directors."
```
Comment (duncathan) [inline code/modules/cargo/packs.dm]: Warranty*

**Review by duncathan — CHANGES_REQUESTED** [review]
mostly just spelling or formatting mistakes. you've done a great job capturing the tone imo, well done!

File: code/modules/cargo/packs.dm (hunk 37 above)
Comment (MrDoomBringer) [reply code/modules/cargo/packs.dm]: yeaaah buddy

File: code/modules/cargo/packs.dm (hunk 25 above)
Comment (duncathan) [reply code/modules/cargo/packs.dm]: you missed this one

File: code/modules/cargo/packs.dm [hunk 41]
```diff
@@ -1888,6 +1904,7 @@
 
 /datum/supply_pack/misc/formalwear
 	name = "Formalwear Crate"
+	desc = "You're gonna like the way you look, I guarentee it. Contains an asston of fancy clothing."
```
Comment (duncathan) [inline code/modules/cargo/packs.dm]: I missed this one

**Review by duncathan — CHANGES_REQUESTED** [review]

**Comment by MrDoomBringer** [comment]
image

**Review by duncathan — APPROVED** [review]

**Pull request merged**